pycaret/classification/oop.py

```diff
@@ -602,6 +607,12 @@ def setup(
             Ignored when ``log_experiment`` is False.
 
 
+        engines: Optional[Dict[str, str]] = None
+            The engine to use for the models, e.g. for lr, users can
+            switch between "sklearn" and "sklearnex" by specifying
+            engines={"lr": "sklearnex"}
```

Comment:
```suggestion
        engines: Optional[Dict[str, str]] = None
            The execution engines to use for the models in the form of a dict
            of `model_id: engine` - e.g. for Logistic Regression ("lr", users can
            switch between "sklearn" and "sklearnex" by specifying
            `engines={"lr": "sklearnex"}`
```

Please update in other places